packages/react/src/hooks/useSwapDriver.ts

```diff
@@ -287,18 +290,49 @@ export const useSwapDriver = ({
             });
           },
           async postInstructions({isLast, amount, isBuy}: IPostInstructionArgs): Promise<InstructionResult<null>> {
-            if (!isLast || !childEntangler || !parentEntangler || !isBuy || !fungibleEntanglerSdk) {
+            if (!isLast || !childEntangler || !parentEntangler || !fungibleEntanglerSdk) {
```

Comment:
Shouldn't we also close intermediary on sellback?

Reply:
Yeah it's doing that